src/redux/sagas/utils/getDomainMetadataChangelog.ts

```diff
@@ -0,0 +1,29 @@
+import { DomainColor, DomainMetadataChangelogInput } from '~gql';
+import { DomainMetadata } from '~types';
+import { excludeTypenameKey } from '~utils/objects';
+
+export const getDomainMetadataChangelog = (
```

Comment:
Small one but maybe name this `getUpdatedDomainMetadataChangelog`, since it's augmenting the current changelog on the metadata obj